src/plugins/intel_cpu/tests/functional/single_layer_tests/adaptive_pooling.cpp

```diff
@@ -162,7 +162,9 @@ class AdaPoolLayerCPUTest : public testing::WithParamInterface<AdaPoolLayerCPUTe
                     dataPtr[i] = pooledVector[i];
                 }
             } else {
-                    tensor = ov::test::utils::create_and_fill_tensor(funcInput.get_element_type(), targetInputStaticShapes[i], 2560, 0, 256);
+                    tensor = ov::test::utils::create_and_fill_tensor(funcInput.get_element_type(),
```

Comment:
to use `InputGenerateData` arg as an  argument of `create_and_fill_tensor` to avoid of misusing of the structure?

Reply:
You mean use something like that?
```
InputGenerateData in_data;
in_data.range = 2560;
in_data.start_from = 0;
...
tensor = ov::test::utils::create_and_fill_tensor(..., in_data);
```
I will take several days to update all usages of create_and_fill_tensor like that

Reply:
Yeah, correct.
It's not burning PR

Reply:
Unfortunately I cannot finish builders reworking (makeConstant) without this PR.
So I updated all...